### DISCUSSION TITLE
Feature Request: Direct PDF Export

### DISCUSSION THREAD

**Discussion by jessienab** [zim-desktop-wiki/zim-desktop-wiki#1987] [Ideas]
I couldn't find anything in issues, nor here, so I wanted to suggest the idea of implementing a direct PDF output from Zim.

At a low level I'm unsure how this can be done, but currently myself I do the following, depending on the content:

- Export to LaTeX format
- Check everything is as I want with a text editor
- Import to LyX
- Export to PDF 

(I know there has to be a way to not need LyX but this works for me at the moment)

It would additionally be nice to export to PDF using the HTML templates, for example ZeroFiveEight is very pleasing visually, but "Printing" a PDF from an HTML export is clunky.

Just wanted to open a discussion if anyone has suggestions for cleaner or simpler export to PDF from Zim, or ideas relating to direct exporting.
Thanks!

**Comment by introt** [comment]
Going to file this under plugin ideas for now. "Direct" PDF creation is out of scope for the core application, but the LaTeX route could be streamlined by a core plugin.

Exporting to PDF using the HTML templates sounds doable via Zim's command line export & [Pandoc](https://stackoverflow.com/questions/44177555/how-to-convert-html-to-pdf-using-pandoc#44180516) in a custom tool.
